Explorer/Assets/DCL/SDKComponents/CameraControl/MainCamera/Systems/MainCameraSystem.cs

```diff
@@ -121,7 +125,7 @@ private void SetupMainCamera(Entity entity)
         {
             if (!sceneStateProvider.IsCurrent || entity != cameraEntity) return;
 
-            World.Add(entity, new MainCameraComponent());
+            World.Add(entity, new MainCameraComponent(new CRDTEntity(0)));
```

Comment:
What is this "zero" entity?

Reply:
the `PBMainCamera` will have a `0` in its `uint VirtualCameraEntity` representing the "empty entity reference field", that's why when forcing the usage of the constructor with it.

But I just changed it to use a nullable in that property and removed the constructor at https://github.com/decentraland/unity-explorer/pull/1744/commits/f8782f6fcf45582fb83a00148e12e8b5f6d27aee